tools/scripts/ansible/cldr-apps-playbook.yml

```diff
@@ -53,6 +53,7 @@
         force: no
         update: no
         version: main
+        remote: origin
```

Comment:
as contrasted with "upstream"? Looks like "origin" is default so this is redundant (not necessarily a bad thing) per https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html

Reply:
it should be, but seemed to fail until this was set.